HomeWork_09/HW9.py

```diff
@@ -0,0 +1,12 @@
+
+text = int(input("Enter a digit: "))
+def my_generator():
+    f1, f2 = 0, 1
+    for i in range(text):
```

Comment:
You shouldn't use variables from the outer scope of your generator. You can pass it as an argument to your `my_generator`

Reply:
Pay attention to this comment as well